beacon-chain/db/deposit_contract.go

```diff
@@ -30,6 +31,11 @@ func (db *BeaconDB) DepositContractAddress(ctx context.Context) ([]byte, error)
 	return addr, nil
 }
 
+// SaveDepositContractAddress is deprecated - use the kv store in beacon-chain/db/kv instead.
```

Comment:
If you add the comment 
```
// DEPRECATED: reason
```
Some IDEs (like goland) will strikethrough the deprecated methods and help the user avoid using them.